pkg/services/app/add.go

```diff
@@ -692,14 +717,136 @@ func (a *App) createPullRequestToRepo(params AddParams, gitProvider gitproviders
 	}
 
 	userRepoRef := gitproviders.NewUserRepositoryRef(github.DefaultDomain, owner, repoName)
-	prLink, err := gitProvider.CreatePullRequestToUserRepo(userRepoRef, params.Branch, appHash, files, utils.GetCommitMessage(), fmt.Sprintf("wego add %s", params.Name), fmt.Sprintf("Added yamls for %s", params.Name))
+	prLink, err := gitProvider.CreatePullRequestToUserRepo(userRepoRef, info.Spec.Branch, appHash, files, utils.GetCommitMessage(), fmt.Sprintf("wego add %s", info.Name), fmt.Sprintf("Added yamls for %s", info.Name))
 	if err != nil {
 		return fmt.Errorf("unable to create pull request: %w", err)
 	}
 	a.logger.Println("Pull Request created: %s\n", prLink.Get().WebURL)
 	return nil
 }
 
+func (a *App) getAppResourceInfo(app wego.Application, clusterName string) *AppResourceInfo {
```

Comment:
Can this just be a pure function with no receiver? It might signal the intent better.

Reply:
Good point. An earlier iteration was referencing something from the app and I neglected to make that change once it no longer did